samples/bootloader/src/main.c

```diff
@@ -59,15 +59,15 @@ static void boot_from(const struct fw_info *fw_info)
 	}
 
 #if !(defined(CONFIG_SOC_NRF9160) || defined(CONFIG_SOC_NRF5340_CPUAPP))
-	/* Protect provision page after firmware is validated so invalidation
-	 * of public keys can be written directly into the page. Note that for
-	 * nRF91, the provision page is kept in OTP which does not need or
-	 * support protection.
+	/* Protect bootloader storage data after firmware is validated so
+	 * invalidation of public keys can be written into the page if needed.
+	 * Note that for nRF91, the bootloader storage data is kept in OTP which
```

Comment:
`for some device, e.g. nRF91 and nRF5340`

Reply:
Done